canonical/ubuntu.com
# Wd 10378 update references to credentials canonical com to skills canonical com

## Done

- Update Emails for Credentials 
- Update style for schedule confirm page 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Check emails have changed
- Try to schedule an exam
    - The confirmation page should use the new suru

## Issue / Card

Fixes [#WD-10378](https://warthogs.atlassian.net/browse/WD-10378)

## Screenshots

[If relevant, please include a screenshot.]


## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
